python/cudf/cudf/_libxx/binaryop.pyx

```diff
@@ -154,14 +154,10 @@ cdef binaryop_s_v(Scalar lhs, Column rhs,
 
 
 def handle_null_for_string_column(Column input_col, op):
-    if op in (
-        BinaryOperation['EQ'], BinaryOperation['LT'],
-        BinaryOperation['LE'], BinaryOperation['GT'],
-        BinaryOperation['GE']
-    ):
+    if op.name.lower() in ('eq', 'lt', 'le', 'gt', 'ge'):
```

Comment:
Just a minor nit here: let's make the input to the function a str, just for consistency with `binop` below.